src/pymordemos/elliptic2.py

```diff
@@ -6,18 +6,23 @@
 """Simple demonstration of solving the Poisson equation in 2D using pyMOR's builtin discretizations.
 
 Usage:
-    elliptic2.py [--fv] PROBLEM-NUMBER N
+    elliptic2.py [--fv] PROBLEM-NUMBER N PRODUCT-COUNT
 
 Arguments:
-    PROBLEM-NUMBER    {0,1}, selects the problem to solve
-    N                 Triangle count per direction
+    PROBLEM-NUMBER  {0,1}, selects the problem to solve
+    N               Triangle count per direction
+
+    PRODUCT-COUNT   0: equip the model with standard products
+                    1: use a fixed parameter instance with random parameter values to also
+                    add an energy product to the model
```

Comment:
Ok, so this is only a very basic demo, but this parameter still feels strange to me. Maybe replace by a `NORM` argument, which is either 'h1' or 'l2' or a number which is taken as the parameter for the energy norm?